command/cp_test.go

```diff
@@ -0,0 +1,62 @@
+package command
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGuessContentType(t *testing.T) {
+	t.Parallel()
+
+	testcases := []struct {
+		filename string
+		content  string
+
+		expectedContentType string
+	}{
+		{
+			filename:            "*.pdf",
+			expectedContentType: "application/pdf",
+		},
+		{
+			filename:            "*.css",
+			expectedContentType: "text/css; charset=utf-8",
+		},
+		{
+			filename: "index",
+			content: `
+					<!DOCTYPE html>
+					<html>
+						<head>
+							<title>Hello World</title>
+						</head>
+						<body>
+							<p>Hello, World! I am s5cmd :)</p>
+						</body>
+					</html>
+					`,
+			expectedContentType: "text/plain; charset=utf-8",
```

Comment:
I'm surprised to see `text/plain` here instead of `text/html`. Please take a look at https://github.com/peak/s5cmd/blob/master/e2e/cp_test.go#L657.

Reply:
thanks, yes it should be `text/html` rather than `text/plain`.